samples/snippets/src/test/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSampleTest.java

```diff
@@ -0,0 +1,109 @@
+import java.util.concurrent.TimeoutException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CreateBatchPredictionJobTextEntityExtractionSampleTest {
+  private static final String PROJECT = System.getenv("UCAIP_PROJECT_ID");
+  private static final String MODEL_ID = System.getenv("TEXT_ENTITY_MODEL_ID");
+  private static final String GCS_SOURCE_URI =
+      "gs://ucaip-samples-test-output/inputs/batch_predict_TEN/ten_inputs.jsonl";
+  private static final String GCS_OUTPUT_URI = "gs://ucaip-samples-test-output/";
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+  private PrintStream originalPrintStream;
+  private String batchPredictionJobId;
+
+  private static void requireEnvVar(String varName) {
+    String errorMessage =
+        String.format("Environment variable '%s' is required to perform these tests.", varName);
+    assertNotNull(errorMessage, System.getenv(varName));
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
+    requireEnvVar("UCAIP_PROJECT_ID");
+    requireEnvVar("TEXT_ENTITY_MODEL_ID");
+  }
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    originalPrintStream = System.out;
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown()
+      throws InterruptedException, ExecutionException, IOException, TimeoutException {
+    CancelBatchPredictionJobSample.cancelBatchPredictionJobSample(PROJECT, batchPredictionJobId);
+
+    // Assert
+    String cancelResponse = bout.toString();
+    assertThat(cancelResponse).contains("Cancelled the Batch Prediction Job");
+    TimeUnit.MINUTES.sleep(2);
+
+    // Delete the Batch Prediction Job
+    DeleteBatchPredictionJobSample.deleteBatchPredictionJobSample(PROJECT, batchPredictionJobId);
+
+    // Assert
+    String deleteResponse = bout.toString();
+    assertThat(deleteResponse).contains("Deleted Batch");
+    System.out.flush();
+    System.setOut(originalPrintStream);
+  }
+
+  @Test
+  public void testCreateBatchPredictionJobTextEntityExtractionSample() throws IOException {
+    // Act
+    String batchPredictionDisplayName =
+        String.format(
+            "temp_java_create_batch_prediction_TEN_%s",
+            UUID.randomUUID().toString().replaceAll("-", "_").substring(0, 26));
+
+    CreateBatchPredictionJobTextEntityExtractionSample
+        .createBatchPredictionJobTextEntityExtractionSample(
+            PROJECT,
+            "us-central1",
+            batchPredictionDisplayName,
+            MODEL_ID,
+            GCS_SOURCE_URI,
+            GCS_OUTPUT_URI);
+
+    // Assert
+    String got = bout.toString();
+    assertThat(got).contains(batchPredictionDisplayName);
+    assertThat(got).contains("response:");
+    batchPredictionJobId = got.split("name:")[1].split("batchPredictionJobs/")[1].split("\"\n")[0];
```

Comment:
Same as above -- Not sure if this is necessary? Is this meant to be printed out or checked?

Reply:
I moved it to the `tearDown()`.